openpype/hosts/unreal/integration/UE_5.0/Source/OpenPype/Private/OpenPypeSettings.cpp

```diff
@@ -0,0 +1,21 @@
+﻿// Fill out your copyright notice in the Description page of Project Settings.
+
+#include "OpenPypeSettings.h"
+
+#include "IPluginManager.h"
+#include "UObjectGlobals.h"
```

Comment:
At least in UE5.1 I had to change these includes like

```suggestion
#include "Interfaces/IPluginManager.h"
#include "UObject/UObjectGlobals.h"
```

Reply:
Seems strange that it needs to be occasionally that specific. But changing it at to this in my workspace still works. I guess we just need to be specific around those include dirs.